app/teacherhome/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 
 from flask import Blueprint
```

Comment:
•	Blueprints is a nice way to organize the application in smaller parts or components, since its not an app with a simple functionality, it requires rbac (Role Based Authentication) were every user has access to different access permisions.